nifi-nar-bundles/nifi-iotdb-bundle/nifi-iotdb-processors/src/main/resources/META-INF.services/org.apache.nifi.processor.Processor

```diff
@@ -0,0 +1,15 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This file needs to be moved from `META-INF.services` to `META-INF/services`, NiFi is unable to read it at the current location.

Reply:
Resolved